strings/medieval_replacement.json

```diff
@@ -0,0 +1,317 @@
+{
```

Comment:
Run this whole file through a JSON beautifier (or even just Format Document in VSC I bet), the indentations are all out of whack.